src/api/status/src/server.js

```diff
@@ -26,7 +26,7 @@ const satelliteOptions = {
         connectSrc: [
           "'self'",
           '*.fontawesome.com',
-          `${host.replace(/(^\w+:|^)\/\//, '')}:4000`,
```

Comment:
I'm not too certain this change on L29 is required. We changed the route for :4000/log and :4000/deploy. Here it is just using :4000 and doesn't specify if it's for log or deploy. @humphd  What happens when we go to :4000? I haven't gone down this route yet.....